serving/installing-logging-metrics-traces.md

```diff
@@ -1,14 +1,14 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups: 
-One that uses Elasticsearch, Kibana, Prometheus and Grafana and 
-another that uses Stackdriver, Prometheus and Grafana. See below 
-for installation instructions for these two setups. You can install 
+Knative Serving offers two different monitoring setups:
+One that uses Elasticsearch, Kibana, Prometheus and Grafana and
+another that uses Stackdriver, Prometheus and Grafana. See below
+for installation instructions for these two setups. You can install
 only one of these two setups and side-by-side installation of these two are not supported.
```

Comment:
How do you remove/uninstall a plugin so that you can switch to the other?

For example, I think Elasticsearch is installed by default? How would I remove that plugin so that I can use stackdriver?

Reply:
Added a section for uninstalling.